esp/scm/ws_workunits_struct.ecm

```diff
@@ -244,6 +244,7 @@ ESPStruct [nil_remove] ECLHelpFile
     [min_ver("1.44")] unsigned PID(0);
     [min_ver("1.58")] unsigned minActivityId(0);
     [min_ver("1.58")] unsigned maxActivityId(0);
+    [min_ver("1.94")] bool IsAvailable(true);
```

Comment:
This really belongs in WsTopology as we display the logs in various places, not just for workunits...

Reply:
This is for the ComponentLog (may be used for other ECLHelpFile in future if needed). The ComponentLog function is only for ECL Workunits in cloud environment. For now, the log function in WsTopology is for bare metal only (log files are always available). I think we should add this (and check this) if we add the logging stack support in WsTopology.

Reply:
FYI The errors you see in the ECL Watch WU Details page are coming from the global logging query.

Reply:
For the logs retrieved by ws_logaccess.GetLogs, could we use the existing ws_logaccess.GetLogAccessInfo to decide whether the log stack is available or not? Or add another flag (IsAvailable?) into ws_logaccess.GetLogAccessInfo?